src/utils/account.js

```diff
@@ -16,22 +16,36 @@ export const extractPublicKey = (passphrase) => {
   return undefined;
```

Comment:
These retuened `undefined` are essentially silent failures. I think we should add a logger here so we don't overlook the broken features.

Reply:
Same goes for the other two convertors.